api/v1alpha1/jobset_types.go

```diff
@@ -93,6 +93,7 @@ type ReplicatedJobStatus struct {
 // +kubebuilder:printcolumn:name="Restarts",JSONPath=".status.restarts",type=string,description="Number of restarts"
 // +kubebuilder:printcolumn:name="Completed",type="string",priority=0,JSONPath=".status.conditions[?(@.type==\"Completed\")].status"
 // +kubebuilder:printcolumn:name="Age",JSONPath=".metadata.creationTimestamp",type=date,description="Time this JobSet was created"
+// +genclient
```

Comment:
```suggestion
// +genclient

```

We need a new line.

Reply:
Yeah just noticed this is mentioned in the article as well - I added an empty line and moved the `// +genclient` to the top above the kubebuilder tags, following the pattern from the Kueue PR but still no luck.

Reply:
What a timing! Nice :)